pkgs/by-name/dc/dcd/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDubPackage,
+}:
+
+buildDubPackage rec {
+  pname = "dcd";
+  version = "0.15.2";
+
+  src = fetchFromGitHub {
+    owner = "dlang-community";
+    repo = "DCD";
+    tag = "v${version}";
+    hash = "sha256-dJ4Ql3P9kPQhQ3ZrNcTAEB5JHSslYn2BN8uqq6vGetY=";
+  };
+
+  dubLock = ./dub-lock.json;
+
+  buildPhase = ''
+    runHook preBuild
+
+    dub build --build=release --config=server
+    dub build --build=release --config=client
+
+    runHook postBuild
+  '';
+
+  doCheck = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 bin/dcd-{client,server} -t "$out/bin"
```

Comment:
Why is this in quotes?

Reply:
Because this way bash variable expansion will consider it one string even if $out contained a space character.
In nixpkgs, this almost never happens,
 but I would still like to get into the habit of writing safe bash code.